natives/pom.xml

```diff
@@ -29,12 +29,25 @@ Copyright 2021, Aayush Atharva
     <packaging>pom</packaging>
     <modules>
         <module>linux-x86_64</module>
+        <module>linux-x86_64-centos6</module>

```

Comment:
No, we shouldn't have a special module for centos6. Your suggested changes should target the existing linux-x86_64 module.

Also, the names of those modules match Netty's expected pattern wrt native libs.

Reply:
I've made changes, as suggested to replace existing linux-x86_64 by the centos build

Reply:
I think we should call it `linux-centos6-x86-64` to keep naming constant with architectures. WDYT @slandelle ?

Reply:
No. The binary is generic enough to work fine on more modern distributions, just like Netty's own binaries.